Products/UmbracoForms/Developer/Prepping-Frontend/index.md

```diff
@@ -7,9 +7,9 @@ Simply add those to your template these can be in the head or at the bottom of t
 Easiest way to add the dependencies is to fetch them from a cdn (like http://www.asp.net/ajax/cdn).
 So simply add the following 3 scripts
 
-	<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-2.1.1.min.js"></script>
-	<script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.13.1/jquery.validate.min.js"></script>
-	<script src="http://ajax.aspnetcdn.com/ajax/mvc/5.1/jquery.validate.unobtrusive.min.js"></script>
+	<script src="//ajax.aspnetcdn.com/ajax/jQuery/jquery-2.1.1.min.js"></script>
```

Comment:
Should always be `https://` - protocol-relative URLs are now considered an anti pattern http://www.paulirish.com/2010/the-protocol-relative-url/